### PR TITLE
Update trajectory widget to leverage trajectory player

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
This depend on this Tesseract [PR](https://github.com/ros-industrial-consortium/tesseract/pull/393)

### CODE REVIEW

File: tesseract_ros/tesseract_msgs/msg/Trajectory.msg [hunk 1]
```diff
@@ -1,5 +1,10 @@
 TesseractState tesseract_state
 
+# Tesseract Command Language Instructions
+string instructions
+
+# DEPRECATED should use instructions
```
Comment (mpowelson) [inline tesseract_ros/tesseract_msgs/msg/Trajectory.msg:6]: I'm not sure about this. A JointTrajectory is pretty standard. I suppose as long as the plotter has necessary utilities to convert a JointTrajectory to command language that is fine.

Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_msgs/msg/Trajectory.msg:6]: I am fine with leaving both. I will remove the deprecated.

Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_msgs/msg/Trajectory.msg]: removed.

**Review by mpowelson — APPROVED** [review]
So what does the slider resolution do? Is the plan that this would eventually interpolate between the waypoints now as you operate the slider? If so, that would be pretty neat, however I still think we need to be able to display the actual waypoints from the planner.

**Comment by Levi-Armstrong** [comment]
> So what does the slider resolution do? Is the plan that this would eventually interpolate between the waypoints now as you operate the slider? If so, that would be pretty neat, however I still think we need to be able to display the actual waypoints from the planner.

Yea, it now interpolates between the waypoints and leverage the time provided by the trajectory to play it correctly. I plan to add a drop down where you select the waypoint in the player and should have that added this week.

**Pull request merged**